src/interface/report/Results/PlayerInfo.tsx

```diff
@@ -60,6 +91,10 @@ const PlayerInfo = ({ combatant }: Props) => {
         <div className="player-details-talents">
           <PlayerInfoTalents talents={talents} />
         </div>
+        <div className="player-details-talents">
+          <h3>Conduits </h3>
+          {conduits!.map((conduit) => renderConduit(conduit))}
```

Comment:
```suggestion
          {conduits?.map((conduit) => renderConduit(conduit))}
```

This may not always be available, using ?. avoids situations where React would otherwise crash.